### DISCUSSION TITLE
Welcome to HACKTOBERFEST2021_PATTERN Discussions!

### DISCUSSION THREAD

**Discussion by aritraroy24** [aritraroy24/HACKTOBERFEST2021_PATTERN#65] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alphasingh** [comment]
I cannot see any PR with 'hacktoberfest-accepted' label, when will this be done? Who is responsible for that?
